src/OSPSuite.UI/Views/Importer/ColumnMappingControl.Designer.cs

```diff
@@ -19,6 +19,7 @@ protected override void Dispose(bool disposing)
          {
             components.Dispose();
          }
+         _gridViewBinder.Dispose();
```

Comment:
@abdelr @georgeDaskalakis It is key to dispose the binders. Otherwise you will get memory leaks